### ISSUE TITLE
BFT barriers

### ISSUE THREAD

**Issue by ZonD80** [tobiabocchi/flipperzero-bruteforce#7]
There are BFT barriers in my area, can this brand be bruteforced?

also tested two NICE barriers, didn't work (low and high frequency). Will continue tests during next week

**Comment by tobiabocchi** [comment]
Do you have any remote we could use to study the protocol? Any raw recording? If it uses fixed code it **probably** can be bruteforced. Same question goes for the NICE barriers, those are not tested in the real field yet, so if you could do that it would be of great help.

**Comment by ZonD80** [comment]
I'll try to get one. Will continue to test nice barriers

**Comment by tobiabocchi** [comment]
Yes, let me know when you get your hands on one, so we can start studying its signal

**Comment by rhysperry111** [comment]
There are at least 5 BFT barriers in the area around me. I've yet to see anybody with a remote though, as they all have number pads installed.
